### ISSUE TITLE
SKPhotoBrowser can't be destroied

### ISSUE THREAD

**Issue by tooodooo** [suzuki-0000/SKPhotoBrowser#43]
I use a collectiongView in UItableViewCell.

```
func collectionView(collectionView: UICollectionView, didSelectItemAtIndexPath indexPath: NSIndexPath) {
        
        var images = [SKPhoto]()
        for _url in vo!.photos {
            let photo = SKPhoto.photoWithImageURL(_url)
            photo.shouldCachePhotoURLImage = true // you can use image cache by true(NSCache)
            images.append(photo)
        }
        
        
        let cell = collectionView.cellForItemAtIndexPath(indexPath) as! GroupViewCollectionViewCell
        let originImage = cell.imageView.image // some image for baseImage
        let browser = SKPhotoBrowser(originImage: originImage!, photos: images, animatedFromView: cell)
        browser.initializePageIndex(indexPath.row)
        
        NSNotificationCenter.defaultCenter().postNotificationName("SHOW_PHOTO_BROWER", object: browser)
    }
```

I handle the notification in tableviewcontroller:

`NSNotificationCenter.defaultCenter().addObserver(self, selector:"showPhotoBrower:", name:"SHOW_PHOTO_BROWER", object: nil)`

```
func showPhotoBrower(note:NSNotification){
        weak var browser = note.object as? SKPhotoBrowser
        presentViewController(browser!, animated: true, completion: {})
    }
```

If I click one collectionviewcell and just  close SKPhotoBrowser, it is worked well.
 I watch next photo in SKPhotoBrowser then close it, it can not be destory. 

SKPhotoBrowser:
```
deinit {
        pagingScrollView = nil
        NSNotificationCenter.defaultCenter().removeObserver(self)
    }
```

This function can not be reached.
Where do I use the wrong？

Thx ur project.

**Comment by suzuki-0000** [comment]
@tooodooo  thx opening issue. I haven't try NSNotification, but it would be string ref problem in SKPhotobrowser. I'll try to check it first.

**Comment by suzuki-0000** [comment]
hi @tooodooo, this bug fixed at 1.7.2 or latest master.
you can reopen issue if you have any problem)
